themes/default/content/blog/introducing-azure-native-v2/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Introducing Azure Native 2.0"
+
+date: 2023-06-13T08:00:00-04:00
+
+meta_desc: "Introducing Pulumi Azure Native Provider 2.0: Streamlined, Expanded, and More Powerful than Ever!"
+
+meta_image: meta.png
+
+authors:
+    - monica-rodriguez
+
+tags:
+    - providers
+    - azure
+
+---
+
+{{% notes type="warning" %}}
+The Pulumi Azure Native Provider 2.0 is currently available in beta.
+{{% /notes %}}
+
+We are thrilled to announce the highly anticipated beta release of the Pulumi Azure Native Provider 2.0, a significant upgrade to our infrastructure as code (IaC) solution for Microsoft Azure. This release brings a host of exciting features and improvements that will enhance your experience with managing Azure resources and empower you to build robust and scalable cloud infrastructure more efficiently.
+
+<!--more-->
+
+At Pulumi, we understand the importance of keeping up with the ever-evolving cloud landscape. The Pulumi Azure Native Provider 2.0 represents our commitment to providing you with the best tools to harness the full power of Microsoft Azure while providing you with same-day access to the entire [Azure API surface](https://docs.microsoft.com/en-us/rest/api/azure/). Let's dive into some of the key enhancements in this release:
```

Comment:
What do we mean here by "same-day"? We don't release every day so it's not true that additions to the Azure API spec are available on the same day.

Reply:
TL;DR: same-day is okay.
We discussed this a few times and the truth is that "we can ship our provider on the same day when API specs are published". In reality, the day when specs are published is fairly meaningful (except for major launches, when we can cut a special release if needed): specs may ship after the service is available, or even before it's available, and service deployment takes days-to-weeks across regions anyway. That's too advanced message, so we stick to "same-day" as mostly a marketing message but with substance that "we can when needed".

Reply:
We have the ability to consume and release upstream changes same-day. We don't normally <need> to do same-day deliveries, but can. It's also possible that our release cadence falls on the same-day as upstream changes.